src/main/java/Duke.java

```diff
@@ -51,14 +66,31 @@ public static void main(String[] args) {
                         index = -1;
                     }
                     todoList.markItem(index, false);
+                }else if(func.equals("todo")){
+                    Todo todo = Todo.toTodo(instruction);
+                    if(todo == null){
+                        printError("Wrong todo format");
+                        continue;
+                    }
+                    todoList.addItem(todo);
+                }else if(func.equals("deadline")){
+                    Deadline deadline = Deadline.toDeadline(instruction);
+                    if(deadline == null){
+                        printError("Wrong deadline format");
+                        continue;
+                    }
+                    todoList.addItem(deadline);
+                }else if(func.equals("event")){
+                    Event event = Event.toEvent(instruction);
+                    if(event == null){
+                        printError("Wrong event format");
+                        continue;
+                    }
+                    todoList.addItem(Event.toEvent(instruction));
```

Comment:
'switch' statement would be better since there are many cases you want to separate.